gazebo_plugins/src/gazebo_ros_tricycle_drive.cpp

```diff
@@ -222,17 +222,15 @@ void GazeboRosTricycleDrive::motorController ( double target_speed, double targe
 {
     double applied_speed = target_speed;
     double applied_angle = target_angle;
-    double applied_steering_speed = 0;
 
     double current_speed = joint_wheel_actuated_->GetVelocity ( 0 );
     if ( wheel_acceleration_ > 0 ) {
         double diff_speed = current_speed - target_speed;
         if ( fabs ( diff_speed ) < wheel_speed_tolerance_ ) {
-            applied_speed = target_speed;
-        } else if ( diff_speed < target_speed ) {
-            applied_speed = current_speed + wheel_acceleration_ * dt;
-        } else {
-            applied_speed = current_speed - wheel_deceleration_ * dt;
+            applied_speed = current_speed;
+        } else if ( fabs(diff_speed) > wheel_acceleration_ * dt ) {
+            if(diff_speed > 0){ applied_speed = current_speed - wheel_acceleration_ * dt;}
```

Comment:
Please use the same bracket indent convention as previous code:
~~~
           if (diff_speed > 0) {
             applied_speed = current_speed - wheel_acceleration_ * dt;
           }
~~~